Exercise_A/index.html

```diff
@@ -1,45 +1,206 @@
+         <div class="col-lg-8 text-center">
+         <h2 class="text-white mt-0 pb-2">Seriously, It's This Fucking Template!</h2>
+         <a href="#" class="btn btn-primary btn-xl" id="google">THAT'S AMAZING!</a>
+      </section>
 
-    <!-- 
-      All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
 
-      If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
-      Your custom JavaScript
-     -->
-  </body>
-</html>
+      <!-- Contact Me Information-->
+      <section class="call-me bg-light" id="call-me">
+         <div class="container px-4 px-lg-5">
+         <div class="row gx-4 gx-lg-5 justify-content-center">
+         <div class="col-lg-10 text-center">
+         <h2 id="greytext">My Contact Information</h2>
+         <hr class="linebreaker">
+         <p>Here's my phone number and email address, because hey that's part of the template and why would you not
+            include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in
+            from all the companies wanting my insane website design skills.
+         </p>
+         <p>Honestly this template does look really nice, though.</p>
+         <div class="container px-4 px-lg-5">
+            <div class="row gx-4 gx-lg-5 justify-content-center">
+               <div class="col-lg-6 text-center">
+                  <i class="fa fa-phone fa-3x animated infinite lightSpeedIn delay-2s" aria-hidden="true"></i>
+                  <p>+1-666-420-6969</p>
+               </div>
+               <div class="col-lg-6 text-center">
+                  <i class="fa fa-envelope-o fa-3x animated infinite lightSpeedIn delay-2s" aria-hidden="true"></i>
+                  <p><a href="mailto:holy@stuff.com">holy@stuff.com</a></p>
+               </div>
+            </div>
+         </div>
+      </section>
+
+
+      <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
+      <!-- 
+         All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
+         
+         If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
+         Your custom JavaScript
+         -->
+         
+      <!-- WOW JS -->
+      <script src="./JS/wow.min.js"></script>
+      <script>new WOW().init();</script>
+      <!-- Bootstrap CDN JS -->
+      <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"crossorigin="anonymous"></script>
+      <!-- Smooth Scroll -->
+      <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll@15/dist/smooth-scroll.polyfills.min.js"></script>
+      <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll@15/dist/smooth-scroll.polyfills.min.js"></script>
+   </body>
+</html>
```

Comment:
You've done a great job in recreating the website. The contents are aligned perfectly. You've made good use of the grid system, which makes your page responsive.
You can refer to the answer repo for more details on how the website the created
https://github.com/XccelerateOrg/FTWD_Jan_2022_Ans/tree/master/Web/FTWD_Web_day_04/Exercise_A